client/webserver/live_test.go

```diff
@@ -1612,6 +1612,12 @@ func (c *TCore) UpdateCert(string, []byte) error {
 func (c *TCore) UpdateDEXHost(string, string, []byte, interface{}) (*core.Exchange, error) {
 	return nil, nil
 }
```

Comment:
Do me a favor, and add these changes 

https://github.com/buck54321/dcrdex/compare/8b805d37e403882a0f0732b21e8d481b110b0540...buck54321:ukane-fiat